jdaviz/conftest.py

```diff
@@ -3,19 +3,41 @@
 # get picked up when running the tests inside an interpreter using
 # packagename.test
 
+import numpy as np
 import pytest
-
-from astropy.wcs import WCS
 from astropy import units as u
 from astropy.nddata import StdDevUncertainty
-import numpy as np
+from astropy.wcs import WCS
 from specutils import Spectrum1D
-from jdaviz.configs.specviz.helper import SpecViz
-from jdaviz.configs.imviz.helper import Imviz
+
+from jdaviz import __version__, CubeViz, Imviz, MosViz, SpecViz, Specviz2d
 
 SPECTRUM_SIZE = 10  # length of spectrum
 
-from jdaviz import __version__
+
+@pytest.fixture
+def cubeviz_app():
```

Comment:
Other tests can also now take advantage of these new fixtures but that is out of scope here.